.drone-1.0.yml

```
@@ -0,0 +1,26 @@
+
+kind: pipeline
+name: default
+type: kubernetes
+steps:
+  - name: install
+    image: node:16
+    environment:
+      NPM_AUTH_USERNAME:
+        from_secret: npm_auth_username
+      NPM_AUTH_TOKEN:
+        from_secret: npm_auth_token
+    commands:
+      - npm ci
```

Since there is a test script, it should be run.

```suggestion
      - npm ci
      - npm test
```

---

Have added that, will now work on fixing the 66 broken tests

---

😬

---

Feel free to remove it if it's a pain. I had assumed the tests would be passing.